SampleApp-Source/sample_app/build.gradle

```diff
@@ -9,8 +9,8 @@ android {
         applicationId "com.liveperson.sample.app"
         minSdkVersion 21
         targetSdkVersion 30
```

Comment:
Let compileSdkVersion and targetSdkVersion bump to 31.
@dmytrosokolov @akshay6533

Reply:
@lptle  we should wait for huawei updates.